vms/volunteer/utils.py

```diff
@@ -0,0 +1,19 @@
+from functools import wraps
+from django.shortcuts import render
+from django.http import Http404
+from volunteer.services import get_volunteer_by_id
+
+def vol_id_check(func):
+    @wraps(func)
+    def wrapped_view(request, volunteer_id):
+        vol = getattr(request.user, 'volunteer', hasattr(request.user, 'administrator'))
```

Comment:
@smarshy Because it also checks if the user is an admin. if the user is a volunteer,it returns the volunteer object, if an admin, it returns true, and if not both, false